app.js

```diff
@@ -3,8 +3,9 @@ const bodyParser = require('body-parser')
 const path = require('path')
 require('./config/config.js')
 const app = express()
+const PORT = 9000
 var allowCrossDomain = function (req, res, next) {
-  var devAllowedOrigins = ['http://localhost:3000', 'https://staging.cribblservices.com']
+  var devAllowedOrigins = ['http://localhost:' + PORT, 'https://staging.cribblservices.com']
```

Comment:
This is for the frontend port. `localhost:3000`

Reply:
Ohh. Will make the changes